config/config.go

```diff
@@ -493,6 +493,8 @@ type IsolationRead struct {
 type Experimental struct {
 	// Whether enable the syntax like `auto_random(3)` on the primary key column.
 	AllowAutoRandom bool `toml:"allow-auto-random" json:"allow-auto-random"`
```

Comment:
Will `enable-auto-random` and `enable-expression-index` be more friendly?

Reply:
Yeah, but I think this should be done in future, not in this PR.